core/amber/src/main/scala/edu/uci/ics/texera/web/resource/aiassistant/AiAssistantManager.scala

```diff
@@ -0,0 +1,40 @@
+package edu.uci.ics.texera.web.resource.aiassistant
+import edu.uci.ics.amber.engine.common.AmberConfig
+import java.net.{HttpURLConnection, URL}
+
+object AiAssistantManager {
+  private val aiAssistantConfig = AmberConfig.aiAssistantConfig
+  val assistantType: String = aiAssistantConfig.getString("assistant")
+  val accountKey: String = aiAssistantConfig.getString("ai-service-key")
+  val validAIAssistant: Boolean = assistantType match {
+    case "none" =>
+      false
+
+    case "openai" =>
+      val sharedUrl = "https://api.openai.com/v1"
```

Comment:
What if OpenAi changed the URL?  A safer way is to let the user specify the URL in the configuration, so that we can easily change it without recompiling the code.